package.json

```diff
@@ -10,7 +10,7 @@
     "webpack": "tsx ./development/webpack/launch.ts",
     "webpack:clearcache": "./development/clear-webpack-cache.js",
     "foundryup": "tsx ./test/helpers/foundry/foundryup.ts",
-    "postinstall": "yarn webpack:clearcache && yarn foundryup",
```

Comment:
loose opinion: I think gitlens _might_ be fine, but not postinstall for everyone